src/com/google/enterprise/adaptor/database/MetadataColumns.java

```diff
@@ -62,13 +60,14 @@
       log.fine("element: `" + e + "'");
       e = e.trim(); 
       String def[] = e.split(":", 2);
-      if (2 != def.length) {
-        String errmsg = "expected two parts separated by colon: `" + e + "'";
-        throw new InvalidConfigurationException(errmsg);
+      if (def.length == 1) {
+        String columnName = def[0].trim();
+        if (!columnName.isEmpty()) {
```

Comment:
I don't like quietly dropping invalid entries on the floor. You're also not checking for empty strings in the else, so the behavior is inconsistent.

If we let empty names through here, as we were doing before, verifyColumnNames will now catch empty column names, but nothing will catch empty metadata names, it looks like those will flow throw to the GSA (possibly a bug in the library's Metadata class).